### PR TITLE
DEM-254 Aside name changes

### PR DESCRIPTION
**Pull request by indytechcook**
DEM-254 Aside name changes

this is @tifletcher branch.  It works so i'm submitting the request

@appirio-tech/demandside

### CODE REVIEW

File: src/client/app/submit-work/aside/submit-work-aside.desktop.scss [hunk 1]
```diff
@@ -13,6 +13,7 @@
       position: fixed;
       margin  : $margin-18;
       top     : 0;
+      width   : 220px;
```
Comment (vietnogi) [inline src/client/app/submit-work/aside/submit-work-aside.desktop.scss]: why is this static?  It should be fluid

Comment (indytechcook) [inline src/client/app/submit-work/aside/submit-work-aside.desktop.scss]: cc @tifletcher he worked a bit to try to make it fluid without success.

Comment (vietnogi) [inline src/client/app/submit-work/aside/submit-work-aside.desktop.scss]: Why did you add this?  What was the problem?

File: src/client/app/submit-work/aside/submit-work-aside.jade [hunk 2]
```diff
@@ -28,11 +28,11 @@
     img(src="/submit-work/images/phone.png")
 
   .project
-    .name Project Name XYZ
+    .name Name: {{getName()}}
```
Comment (vietnogi) [inline src/client/app/submit-work/aside/submit-work-aside.jade]: why are these methods instead of scope objects?

File: src/client/app/submit-work/aside/submit-work-aside.desktop.scss (hunk 1 above)
Comment (tifletcher) [inline src/client/app/submit-work/aside/submit-work-aside.desktop.scss]: @vietnogi fixed position defines its width relative to the viewport. when you scroll down and the top bar disappears the aside becomes pos: fixed and so the width of the wrapper becomes dependent on the length of the work request name.

better solution welcome :)

File: src/client/app/submit-work/submit-work.service.js [hunk 3]
```diff
@@ -83,6 +85,23 @@
...
+    function getName() {
+      if (work.name)
+        return work.name;
+      return '';
+    }
+
+    function getRequestType() {
+      if (work.requestType && work.requestType.length > 0) {
+        if (work.requestType == 'both')
+          return 'Design & Code';
```
Comment (vietnogi) [inline src/client/app/submit-work/submit-work.service.js]: `Design & Code` should be kept in the HTML

File: src/client/app/submit-work/submit-work.service.js [hunk 4]
```diff
@@ -83,6 +85,23 @@
...
+        // eww stinky
+        return work.requestType.charAt(0).toUpperCase() + work.requestType.substr(1) + " Only";
```
Comment (vietnogi) [inline src/client/app/submit-work/submit-work.service.js]: `Only` should be kept in HTML

File: src/client/app/submit-work/aside/submit-work-aside.desktop.scss (hunk 1 above)
Comment (vietnogi) [inline src/client/app/submit-work/aside/submit-work-aside.desktop.scss]: Thats not quite accurate, but can you give me the steps to reproduce so I can figure out the root of the problem.

**Comment by indytechcook** [comment]
@vietnogi 100% with the placement of stuff, i'm updating

File: src/client/app/submit-work/aside/submit-work-aside.desktop.scss (hunk 1 above)
Comment (indytechcook) [inline src/client/app/submit-work/aside/submit-work-aside.desktop.scss]: I think his issue was that the size of the aside would change depending on the size of the project name.

Comment (tifletcher) [inline src/client/app/submit-work/aside/submit-work-aside.desktop.scss]: steps to reproduce:
1. clone branch
2. remove this line of css
3. watch the name wrap nicely
4. scroll down until the top bar disappears (pos: fixed on <aside>)

**Comment by vietnogi** [comment]
added a max-width to keep container fluid, everything else lgtm

**Comment by indytechcook** [comment]
@vietnogi just pushed a bunch of updates.

**Comment by tladendo** [comment]
lgtm
